src/index.html

```diff
@@ -6,7 +6,39 @@
     <title>Calendar</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Calendar</h1>
+  <body class ="body">
+    <div class="calendar calendar--start-day--mon calendar--month--30 body__calendar--size-modifier">
```

Comment:
```suggestion
    <div class="calendar calendar--start-day--mon calendar--month--30">
```
Do you really need this class? You don't use it at all